GetMessage/__tests__/handler.test.ts

```diff
@@ -443,181 +431,7 @@ describe("GetMessageHandler", () => {
     );
   });
 
-  it("should respond with forbidden if requesting user is not allowed to see legal message", async () => {
```

Comment:
I would have maintained test on message retrieval, because we still allow GetMessage to have legal data for retrocompatibility.

Reply:
[043f255](https://github.com/pagopa/io-functions-services/pull/278/commits/043f2559321c790c2627335f9e21ad971a52de5b)